Assets/Mirror/Runtime/MessagePacker.cs

```diff
@@ -130,6 +130,12 @@ public static bool UnpackMessage(NetworkReader messageReader, out int msgType)
                 networkMessage.conn.Disconnect();
                 return;
             }
+            finally
+            {
+                // TODO: Figure out the correct channel
+                ProfilerData.Receive(message, Channels.DefaultReliable, networkMessage.reader.Length);
```

Comment:
Can you make the channel number `-1` for now until this is implemented as a temp value?

Reply:
sure